data/mods/petmod/rulesets.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**@type {{[k: string]: ModdedFormatsData}} */
+let BattleFormats = {
+	standardpetmod: {
+		inherit: true,
+		ruleset: ['[Gen 7] OU', '!Species Clause'],
+		banlist: ['All Pokemon'],
+		onBegin() {
+			// The only validator rule this currently modifies is Species Clause, so the added rule is just this
+			this.add('rule', 'Species Clause: Limit one of each Pokémon');
```

Comment:
No, this doesn't work. This is the line client has trouble with. You can't send a message starting with `|rule|Species Clause:`.

Reply:
ok